cornavirus.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Novel Coronavirus 2020
+description: Data and analysis on the Coronavirus Pandemic
+keywords: Coronavirus, Covid-19
+date: 2020-03-21
+modified: 2020-03-21
+image: 
+---
+
+Qu: What are the key figures of Covid-19 worldwise, such as:
```

Comment:
We don't need a question section - that was just in the other one because that was how it was structured. Here you could just start with main section of readme.